packages/ui/src/components/vuestic-components/va-switch/VaSwitch.demo.vue

```diff
@@ -166,7 +166,7 @@
     <VbCard title="Stateless switch without v-model">
```

Comment:
On keyboard navigation I was able to land focus on disabled switch. That should not be the case.
![image](https://user-images.githubusercontent.com/5394573/82702725-3d34c380-9c7b-11ea-8509-5633a554c91f.png)